.gitignore

```diff
@@ -60,3 +60,5 @@ alias.json
 */**/*.log
 
 */**/gh-pages
+
+.prettierrc
```

Comment:
I'd suggest committing this as well, perhaps we can workout a good prettier setup that can be shared across teams.
Please also add ```*.prettierrc``` entry to ```.gitattributes``` as well.

Reply:
done

Reply:
done